package.json

```diff
@@ -15,17 +15,23 @@
   "main": "index.js",
   "types": "tree-sitter.d.ts",
   "dependencies": {
-    "nan": "^2.14.0",
-    "prebuild-install": "^6.0.1"
+    "nan": "^2.17.0",
+    "prebuild-install": "^7.1.1"
   },
   "devDependencies": {
-    "@types/node": "^14.14.31",
-    "chai": "^4.3.3",
-    "mocha": "^8.3.1",
-    "prebuild": "^10.0.1",
-    "superstring": "^2.4.2",
+    "@types/node": "^18.14.6",
+    "chai": "^4.3.7",
+    "mocha": "^10.2.0",
+    "node-gyp": "9.3.1",
```

Comment:
This isn't strictly necessary but it lets you run the project with Python 3.11

See https://github.com/prebuild/prebuild/issues/286#issuecomment-1460871873